applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_jump_process_2d.py

```diff
@@ -28,10 +28,10 @@ def __init__(self, Model, settings):
             raise Exception(err_msg)
         self.body_model_part = Model[body_model_part_name]
 
-        self.velocity_infinity = [0,0,0]
-        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
-        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
-        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
+        self.free_stream_velocity = [0,0,0]
```

Comment:
```suggestion
        self.free_stream_velocity = settings["free_stream_velocity"].GetVector()
```
I think that this should work.

Reply:
This works! I included your suggestion in https://github.com/KratosMultiphysics/Kratos/pull/4668. Since compute_lift_jump_process_2d.py is anyway removed in https://github.com/KratosMultiphysics/Kratos/pull/4668, I will leave it here like this.